views/pages/profile.ejs

```diff
@@ -258,7 +224,118 @@
 
   </body>
 
-  <script defer src="/js/auth.js"></script>
+  <!-- handeling form data -->
```

Comment:
You removed the Auth.js file. please check that script should be kept in /js/ folder in public dir